changelog/undistributed.rst

```diff
@@ -0,0 +1,19 @@
+* Please follow the template we introduced in NOVEMBER.md file.
```

Comment:
We moved the changelog to this folder ```changelog/undistributed```, can you create a new changelog file under this folder. There is a template.rst you can refer to. The file name can be ```216_XE_ShowApConfigGeneral.rst```

https://github.com/CiscoTestAutomation/genieparser/tree/master/changelog/undistributed